tests/pkunit1_test.py

```diff
@@ -10,6 +10,17 @@ def test_case_dirs():
     from pykern import pkunit
 
     for d in pkunit.case_dirs():
+        if d.basename == 'xlsx':
```

Comment:
Don't overlap the case inputs. case_dirs could test excel, too, because it can write an xlsx, but that's not really necessary since it uses file_eq.